docs/modules/ROOT/pages/foundry/pages/foundry-defender.adoc

```diff
@@ -0,0 +1,156 @@
+= OpenZeppelin Defender integration
+
+OpenZeppelin Foundry Upgrades can be used for performing deployments through https://docs.openzeppelin.com/defender/[OpenZeppelin Defender], which allows for features such as gas pricing estimation, resubmissions, and automated bytecode and source code verification.
+
+WARNING: Defender deployments are **always** broadcast to a live network, regardless of whether you are using the `broadcast` cheatcode.
+The recommended pattern is to separate Defender scripts from scripts that rely on network forking and simulations, to avoid mixing simulation and live network data.
+
+== Installation
+
+See xref:foundry-upgrades#installion[Using with Foundry - Installation].
+
+== Prerequisites
+1. Install https://nodejs.org/[Node.js].
+
+2. Configure your `foundry.toml` to include build info and storage layout:
+[source,toml]
```

Comment:
A couple of code blocks in the generated docs show up with white background:

<img width="770" alt="Captura de pantalla 2024-03-08 a la(s) 7 36 25 p m" src="https://github.com/OpenZeppelin/openzeppelin-upgrades/assets/33379285/3ff7663c-ce71-4ad0-9aa8-4c40e3df722a">

I guess it's defaulting if it doesn't recognize the format (assuming it's not recognizing `toml`). Not a blocker but it may be good to review it in https://github.com/OpenZeppelin/openzeppelin-foundry-upgrades

Reply:
Thanks! Opened https://github.com/OpenZeppelin/openzeppelin-foundry-upgrades/issues/27 to address this separately.